db/models/topics.models.js

```diff
@@ -140,6 +140,23 @@ exports.changeVotesById = (article_id, votesToChange) => {
       } else {
         return PromResults[1].rows;
       }
-    
   });
 };
+
+exports.removeCommentAtId = (comment_id) => {
+  let queryString = `DELETE FROM comments `;
+  const queryValues = [];
+  const queryPromises = [];
+  queryString += ` WHERE comment_id = $1 RETURNING * `
+  queryValues.push(comment_id);
+  queryPromises.push(checkExists("comments", "comment_id", comment_id));
```

Comment:
Using check exists is necessary here - we can infer that the comment doesn't exist if 0 rows are returned from out deletion query (since you have a RETURNING * on it)